sentry-ruby/lib/sentry/client.rb

```diff
@@ -7,7 +7,9 @@ class Client
 
     def initialize(configuration)
       @configuration = configuration
-      @transport = case configuration.scheme
+      scheme = configuration.scheme || configuration.dsn.scheme
```

Comment:
I'll remove the scheme concept from configuration in another PR, along with other Transport options